backend/endpoints/jokes/blueprint.py

```diff
@@ -1,11 +1,72 @@
-from flask import Blueprint, make_response
+from flask import Blueprint, make_response, request
+from utils.myclass import EelJokes, db
+import json
 
 bp = Blueprint(
     "eelbot_joke",
     __name__,
     url_prefix="/joke",
 )
 
-@bp.route("/")
-def main():
-    return make_response(("Hello Eels!", 200))
\ No newline at end of file
+# @bp.route("/")
+# def main():
+#     return make_response(("Hello Eels!", 200))
```

Comment:
```suggestion
```

Delete unused code

Reply:
This is one of those 'clean code' principles. The best thing to do is avoid commented out sections of code. If it's required, implement it, otherwise get rid of it. Feel free to keep around helpful commented code locally, but don't stage and commit them.